config-controller/api/v1alpha1/policy_types.go

```diff
@@ -261,8 +275,12 @@ func (p SecurityPolicySpec) ToProtobuf(notifiers map[string]string) (*storage.Po
 	}
 
 	for _, scope := range p.Scope {
+		cluster, err := getNameFromIdOrName(scope.Cluster, caches[Cluster])
```

Comment:
Suggestion for better readability:
Have getNotifierID (notifierName string) and getClusterID (clusterName string) funcs which in turn calls getID function.